include/eigenpy/details.hpp

```diff
@@ -93,11 +93,12 @@ namespace eigenpy
     NumpyType()
     {
       pyModule = bp::import("numpy");
+      // TODO I don't know why this Py_INCREF is necessary.
+      // Without it, the destructor of NumpyType SEGV sometimes.
+      Py_INCREF(pyModule.ptr());
       
       NumpyMatrixObject = pyModule.attr("matrix");
       NumpyMatrixType = reinterpret_cast<PyTypeObject*>(NumpyMatrixObject.ptr());
-      NumpyAsMatrixObject = pyModule.attr("asmatrix");
```

Comment:
Why have you deleted that? Is it deprecated on newer versions of NumPy?

Reply:
It isn't used so I thought it was here as a result of development code not being cleaned up.

Reply:
I don’t agree as part of the current users are still using Python 2. We cannot deprecate now this feature even if Python will be over soon.

Reply:
I would prefer to keep as they are not related to the current bug.